test/specs/collections/Breadcrumb/BreadcrumbDivider-test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import BreadcrumbDivider from 'src/collections/Breadcrumb/BreadcrumbDivider'
+import * as common from 'test/specs/commonTests'
+
+describe('BreadcrumbDivider', () => {
+  common.isConformant(BreadcrumbDivider)
+  common.rendersChildren(BreadcrumbDivider)
+
+  it('renders as a div by default', () => {
+    shallow(<BreadcrumbDivider />)
+      .should.have.tagName('div')
+  })
+
+  describe('icon prop', () => {
```

Comment:
See [#321 (comment)](https://github.com/TechnologyAdvice/stardust/pull/321/files#r70022927) for `common.implementsIconProp()`.